### PR TITLE
Parse BRW Stats

### PR DESCRIPTION
**Pull request by roclark**
The Block Read/Write (BRW) stats file includes the following (paraphrased from [the Lustre manual](http://wiki.old.lustre.org/manual/LustreManual20_HTML/LustreProc.html#50438271_55057))

- Pages per block R/W: Counter value of the number of pages per RPC (remote procedure call)
- Discontiguous pages: Counter value of the number of logical discontinuities per RPC
- I/O Time: The time in milliseconds the filesystem has spent processing various object sizes
- Disk I/O Size: Counter value of the number of operations the filesystem has performed for the given size
- Disk I/O's in Flight: Current number of I/O operations that are processing during the snapshot

### CODE REVIEW

**Comment by roclark** [comment]
@joehandzik this is a pretty big refactor, so please let me know if you want me to change anything and I will be glad to do so.

File: sources/procfs.go [hunk 1]
```diff
@@ -63,6 +63,7 @@ func (s *lustreSource) generateOSSMetricTemplates() error {
 		"obdfilter/*": map[string]string{
 			"blocksize":            "Filesystem block size in bytes",
 			"brw_size":             "Block read/write size in bytes",
+			"brw_stats":            "A collection of block read/write statistrics",
```
Comment (joehandzik) [inline sources/procfs.go]: *statistics

**Comment by joehandzik** [comment]
It looks pretty good so far, but I'll come back to it a few times today to see if I have any useful suggestions. Clearly, some of these files are a pain to parse.

**Comment by roclark** [comment]
Sounds good! Agreed. But in the case with this file, the reward is very much worth it as I'm able to scrape some really useful data about the cluster.

**Comment by roclark** [comment]
Help text is added and it compiles and runs as expected!

**Comment by joehandzik** [comment]
LGTM

**Pull request merged**